### ISSUE TITLE
Cannot start game for version 16.0

### ISSUE THREAD

**Issue by robertmiszczuk** [anegostudios/VintageStory-Issues#1619]
**Game Version:** 16.0
**Platform:** Linux 
**Modded:** No
**SP/MP:** Singleplayer

### Description
Game crashed for me after installing 16.0
no mods / I removed previous version and copied game file into empty folder (the same as previous)

### Logs
```
Running on 64 bit Linux with 15 GB. 
Version: v1.16.0 (Stable)20.01.2022 19:30:42: Critical error occurred
System.ArgumentException: An item with the same key has already been added. Key: xlib
  at System.Collections.Generic.Dictionary`2[TKey,TValue].TryInsert (TKey key, TValue value, System.Collections.Generic.InsertionBehavior behavior) [0x0015a] in <533173d24dae460899d2b10975534bb0>:0 
  at System.Collections.Generic.Dictionary`2[TKey,TValue].Add (TKey key, TValue value) [0x00000] in <533173d24dae460899d2b10975534bb0>:0 
  at System.Linq.Enumerable.ToDictionary[TSource,TKey] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] keySelector, System.Collections.Generic.IEqualityComparer`1[T] comparer) [0x00086] in <5b415632df1f4365ae2242b1a257bb5b>:0 
  at System.Linq.Enumerable.ToDictionary[TSource,TKey] (System.Collections.Generic.IEnumerable`1[T] source, System.Func`2[T,TResult] keySelector) [0x00000] in <5b415632df1f4365ae2242b1a257bb5b>:0 
  at Vintagestory.Common.ModLoader.CheckAndSortDependencies (System.Collections.Generic.IEnumerable`1[T] mods) [0x00064] in <d16f4e016eef4b52a1d7221d4e54ddfc>:0 
  at Vintagestory.Common.ModLoader.verifyMods (System.Collections.Generic.List`1[T] mods) [0x00007] in <d16f4e016eef4b52a1d7221d4e54ddfc>:0 
  at Vintagestory.Common.ModLoader.DisableAndVerify (System.Collections.Generic.List`1[T] mods, System.Collections.Generic.IEnumerable`1[T] disabledModsByIdAndVersion) [0x00015] in <d16f4e016eef4b52a1d7221d4e54ddfc>:0 
  at Vintagestory.Common.ModLoader.LoadModInfosAndVerify (System.Collections.Generic.IEnumerable`1[T] disabledModsByIdAndVersion) [0x00007] in <d16f4e016eef4b52a1d7221d4e54ddfc>:0 
  at _vFhcDU5n6Z0hUte90jdaxlrDjR5._stt3ckg04XAvVNS4fr54wm89CXe () [0x00065] in <d16f4e016eef4b52a1d7221d4e54ddfc>:0 
  at _vFhcDU5n6Z0hUte90jdaxlrDjR5._hndzNoMYcegNh6f26ARDQKxU9eE () [0x00000] in <d16f4e016eef4b52a1d7221d4e54ddfc>:0 
  at Vintagestory.API.Common.TyronThreadPool+<>c__DisplayClass3_0.<QueueTask>b__0 (System.Object a) [0x00000] in <36cbe087d5194ceb880973ce6f2dfbfa>:0 
  at System.Threading.QueueUserWorkItemCallback.WaitCallback_Context (System.Object state) [0x00007] in <533173d24dae460899d2b10975534bb0>:0 
  at System.Threading.ExecutionContext.RunInternal (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00071] in <533173d24dae460899d2b10975534bb0>:0 
  at System.Threading.ExecutionContext.Run (System.Threading.ExecutionContext executionContext, System.Threading.ContextCallback callback, System.Object state, System.Boolean preserveSyncCtx) [0x00000] in <533173d24dae460899d2b10975534bb0>:0 
  at System.Threading.QueueUserWorkItemCallback.System.Threading.IThreadPoolWorkItem.ExecuteWorkItem () [0x00021] in <533173d24dae460899d2b10975534bb0>:0 
  at System.Threading.ThreadPoolWorkQueue.Dispatch () [0x00074] in <533173d24dae460899d2b10975534bb0>:0 
  at System.Threading._ThreadPoolWaitCallback.PerformWaitCallback () [0x00000] in <533173d24dae460899d2b10975534bb0>:0 
```

**Comment by copygirl** [comment]
Bug has been fixed for the upcoming version, see #1601.

You mention you have no mods, but the error mentions the mod XLib. Are you sure there's not still mods in `%AppData%/VintagestoryData/Mods` or so?

**Comment by robertmiszczuk** [comment]
Ok, the mods were hidden in another folder - not the game folder. Weird but after many reinstalls of newer versions I guess it can happen. Sorry for the confusion
